content/pt/docs/referencia-da-api/kyc/_index.md

```diff
@@ -7,5 +7,8 @@ weight: 14
 description: >
     Conheça os endpoints para realização do KYC de seu cliente.
 ---
+<BR>
+
+Atualmente os endpoints de KYC só podem ser utilizados em casos especiais. Caso seja de seu interesse, contate nosso time comercial através desse <a href="https://app.pipefy.com/public/form/Qz4ptt_W/?origem_do_lead=Documenta%C3%A7%C3%A3o" target="_blank">formulário </a>.
```

Comment:
wouldn't be better to keep with the markdown syntax?

```suggestion
Atualmente os endpoints de KYC só podem ser utilizados em casos especiais. Caso seja de seu interesse, contate nosso time comercial através desse [formulário](https://app.pipefy.com/public/form/Qz4ptt_W/?origem_do_lead=Documenta%C3%A7%C3%A3o).
```

Reply:
I would like the link to open on another page as it does not refer to a documentation link. Is it possible on markdown?

Reply:
ohhh, got it. Currently not, just on recent versions of markdown, but it seems Hugo doesn't implement them already